src/Engines/thinEngine.ts

```diff
@@ -171,6 +171,29 @@ export class ThinEngine {
         Effect.ShadersRepository = value;
     }
 
+     /**
+     * Gets or sets the textures that the engine should not attempt to load as compressed
+     */
+    public static ExcludedCompressedTextures: string[] = [];
+
+    /**
+     * Filters the compressed texture formats to only include
+     * files that are not included in the skippable list
+     *
+     * @param url the current extension
+     * @param textureFormatInUse the current compressed texture format
+     * @returns "format" string
+     */
+    public static ExcludedCompressedTextureFormats(url: Nullable<string>, textureFormatInUse: Nullable<string>): Nullable<string> {
+        const skipCompression = (): boolean => {
+            return ThinEngine.ExcludedCompressedTextures.some((entry) => {

```

Comment:
not sure we should use some for older browser support as we do not enforce polyfill use ???

Reply:
I can do that, but I think the Typescript transpiler should handle that. But it seems like browser support for it is pretty universal:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

Can you confirm?

Reply:
Looks like it is already everywhere we need it so you are fine :-)